src/git.rs

```diff
@@ -0,0 +1,24 @@
+use git2::{build::CheckoutBuilder, build::RepoBuilder, Repository as GitRepository,
+           RepositoryInitOptions};
+use quicli::prelude::*;
+use std::fs::remove_dir_all;
```

Comment:
wait why are we no longer using the remove_dir_all crate?

Reply:
oh crap. mistakes were made. thanks for catching!